models/upload.go

```diff
@@ -0,0 +1,155 @@
+// Copyright 2016 The Gogs Authors. All rights reserved.
```

Comment:
Please update to current year and `Gitea`

Reply:
Hi. I've added a Gitea attribution to that file. The remaining code in there is basically repo_editor.go so should probably keep the Gogs attribution.